static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -516,13 +510,24 @@ const RepeatedContent = styled(LeftLineTitle)`
 `;
 
 const DefaultLine = styled('div')<{
-  hasToggle: boolean;
+  isExpandable: boolean;
+  isExpanded: boolean;
   isSubFrame: boolean;
 }>`
+  position: relative;
   display: flex;
   justify-content: space-between;
   align-items: center;
-  background: ${p => (p.isSubFrame ? `${p.theme.surface100}` : '')};
+  background: ${p => (p.isSubFrame ? `${p.theme.surface100}` : `${p.theme.surface200}`)};
+  min-height: 32px;
+  word-break: break-word;
```

Comment:
These new styles are necessary because I removed `className="title"`. I needed to remove the legacy CSS because I wanted to remove the border-radius